docs/src/modules/components/Ad.js

```diff
@@ -53,7 +53,15 @@ const inHouseAds = [
     name: 'scaffoldhub',
     link: 'https://scaffoldhub.io/?partner=1',
     img: '/static/in-house/scaffoldhub.png',
-    description: '<b>ScaffoldHub</b> - Automate building your full-stack Material-UI web-app.',
+    description: '<b>Scaffold</b><br />Automate building your full-stack Material-UI web-app.',
```

Comment:
```suggestion
    description: '<b>ScaffoldHub</b><br />Automate building your full-stack Material-UI web-app.',
```

Reply:
The logo is branded, I wanted to cover the generic noun. What do you think?